galaxy/normalization/NmrNormalization_wrapper.R

```diff
@@ -22,6 +22,24 @@ options(stringsAsFactors = FALSE)
 # For parseCommandArgs function
 library(batch) 
 
+# Constants
+argv <- commandArgs(trailingOnly = FALSE)
+script.path <- sub("--file=","",argv[grep("--file=",argv)])
+prog.name <- basename(script.path)
+
+# Print help
+if (length(grep('-h', argv)) >0) {
+	cat("Usage:", prog.name,
+	    "dataMatrix myDataMatrix.tsv",
+	    "scalingMethod PQN|QuantitativeVariable",
+	    "graphType None|Overlay|One_per_individual",
+	    "logOut myLog.txt",
+	    "dataMatrixOut myDataMatrixOutput.tsv",
+	    "graphOut myGraph.pdf",
+		"\n")
+	quit(status = 0)
+}
+
```

Comment:
Maybe this part should be move to easyrlibrary if all our scripts have to present an `-h` option ?

Reply:
Note that it isn't mandatory to merge this PR

Reply:
But the help message is different in each tool.